inst/EpiModel 2.0 Introduction.rmd

```diff
@@ -0,0 +1,254 @@
+---
+title: 'EpiModel 2.0: Introduction to new EpiModel Workflow'
+author: "EpiModel Research Lab"
+date: "May 4, 2020"
+output:
+  html_document:
+    toc: yes
+    toc_float: yes
+  word_document:
+    toc: yes
+---
+
+```{r echo = FALSE, eval = TRUE}
+suppressMessages(suppressWarnings(library(EpiModel)))
+```
+
+# Introduction
+
+The newest release of EpiModel, dubbed "EpiModel 2.0", brings with it several changes to the EpiModel workflow you may be accustomed to from previous version of EpiModel, as well as adding new functionality to the platform in order to make running dynamic infectious disease models even easier. In this tutorial, we will go through the major changes to EpiModel, detail the new features that have been added and in the final section provide examples on how to migrate your code from pre to post-EpiModel 2.0.
+
+# EpiModel Changes
+
+In this section, we will detail the core functions that remain in EpiModel 2.0, but that received an overhaul for efficiency and end-user accesiblility. All changes listed in this section will be further demonstrated in the "EpiModel 1.0: Code Migration" section of this document.
+
+## Mode to Group
+
+The most significant change to the EpiModel workflow is in how we handle heterogenous sub-populations. In the original EpiModel worfklow, designation of a second population under interest, was done by initializing a new network and setting it's ```bipartite``` input parameter equal to the number of actors in the first mode:
```

Comment:
Just use single backticks for in-line code

Reply:
Fixed.